templates/profile/profile.html.twig

```diff
@@ -3,7 +3,15 @@
     <i class="fa fa-pencil" aria-hidden="true"></i>  {% trans from 'messages' %}basics.edit{%  endtrans %}
   </a>
 </div>
+      
 
+{% if (user.isSuperAdmin) %}
+  <p><strong>{% trans %}profile.tags{% endtrans %}</strong>:
+    {% for tag in user.customer.tags|expand_tags %}
+      <span class="label label-default" style="background-color: {{ tag.color }};">{{ tag.slug }}</span>
+    {% endfor %}
+  </p>
+{% endif %}  
```

Comment:
- It's useless for admins to know about their own tags
- We are not using the "super admin" role in any case